scripts/bump-version/index.js

```diff
@@ -116,6 +113,11 @@ const updateDeps = (pkgJson) => {
     })
 }
 
+const listAllVersions = () => {
+    sh.echo('\nVersions of packages in the monorepo:\n')
+    sh.exec('lerna list --all --long')
```

Comment:
fyi, using `lerna` directly rather than `npx lerna` or whatever means you can't `node scripts/bump-version`, you have to use `npm run bump-version` (or install `lerna` globally, but that's gross).